pages/education.md

```diff
@@ -1,38 +1,71 @@
 title: The Education Track
 ---
-Since 2011 we have welcomed teachers to PyCon UK's special education track
-to promote programmer / teacher collaboration and mutual learning.
-Since 2012 we have put on a kids' day where the proto-programmers of tomorrow
-explore Python with the technologists of today.
-
-We're doing it again this year,
-on Saturday 28th October at Cardiff City Hall.
-
-Teachers: come learn about Python,
-its vibrant community and its uses in education;
-"adopt a developer" and create new educational resources;
-meet colleagues from the Raspberry Pi Foundation, Micro:bit Foundation and Python Software Foundation;
-and observe experts giving lessons to kids in the kids' day.
-
-Kids (and parents): come learn how to build the future at PyCon UK.
-Experts in computing education and professional programmers will be on hand
-to deliver a series of fun, practical and interesting workshops
-that'll help you learn the basics of programming,
-develop your computing skills and make cool projects.
+Since 2011 we have welcomed teachers to PyCon UK's education track to promote
+programmer / teacher collaboration and mutual learning. Since 2012 we have put
+on a children's day where the proto-programmers of tomorrow explore Python with
+the technologists of today.
+
+We're doing it again this year, on Saturday 28th October at Cardiff City Hall.
+
+Teachers: come learn about Python, its vibrant community and its uses in
+education; "adopt a developer" and create new educational resources; meet
+colleagues from the Raspberry Pi Foundation, Micro:bit Foundation and Python
+Software Foundation; and observe experts giving lessons to kids on the
+children's day.
+
+Kids (and parents): come learn how to build the future at PyCon UK. Experts in
+computing education and professional programmers will be on hand to deliver a
+series of fun, practical and interesting workshops that'll help you learn the
+basics of programming, develop your computing skills and make cool projects.
+
+### What's on
+
+**[Picademy](https://www.raspberrypi.org/training/picademy/)**: the Raspberry Pi Foundation's free face-to-face training programme that supports educators in
+their digital making and computing journey. It's a two-day CPD event held at
```

Comment:
minor: I had to google “CPD event” to know what the acronym stands for, but I’m guessing you expect teachers to understand this?